.eslintrc

```
@@ -8,10 +9,11 @@
   },
   "extends": "eslint:recommended",
   "rules": {
+    "semi": "warn"
   },
   "env": {
     "es6": true,
     "node": true,
     "jest": true
   }
-}
+}
```

Missing new line at the end of the file.

```suggestion
}

```